app-web/src/App.jsx

```diff
@@ -1,9 +1,17 @@
-import ProblemListPage from './components/ProblemListPage';
+import React from 'react';
+import ProblemList from './components/ProblemList';
+import ProblemDetail from './components/ProblemDetail';
+import ProblemForm from './components/ProblemForm';
+
+import { Route } from 'react-router-dom';
```

Comment:
리액트 17버전 이상부터는 `React`를 임포트 하지 않아도 됩니다. 

라이브러리 임포트와 로컬 컴포넌트 임포트를 묶으면 좋을 것 같습니다.

```suggestion
import React from 'react';
import { Route } from 'react-router-dom';

import ProblemList from './components/ProblemList';
import ProblemDetail from './components/ProblemDetail';
import ProblemForm from './components/ProblemForm';
```

See also
  - https://ko.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html